modules/internal/DefaultSparse.chpl

```diff
@@ -424,7 +430,7 @@ module DefaultSparse {
       const (found, loc) = dom.find(ind);
       if found then
         return data(loc);
-      else
+      else // ?fromMMS: is this error message correct? Not actually looking at value.
```

Comment:
If the index is not found, then the value corresponding to that index is *typically* zero in a sparse array. But then this begs the questions:

- What if array has a different IRV (implicitly replicated value)?
- Is this still meaningful given that I can add an index and set it to zero?

We can change the error message, but I don't see a good and concise alternative. And that's probably out of scope of this PR.